pkgs/os-specific/darwin/apple-source-releases/bootstrap_cmds/default.nix

```diff
@@ -3,27 +3,18 @@
 appleDerivation {
   nativeBuildInputs = [ yacc flex ];
 
-  buildPhase = ''
+  patchPhase = ''
     cd migcom.tproj
```

Comment:
Recommend using `sourceRoot = "migcom.tproj"`

Reply:
Not just recommended: overriding the `patchPhase` breaks the `patches`

Reply:
I could use `postPatch`. But setting `sourceRoot` it's little complicated, it has a parent dir `bootstrap_cmds-121`, but our version now is `121-macos-11.0.1`, we need to squeeze out `121`, while `cd` will be easy.

Reply:
If you really need some bash logic, that needs to go to `setSourceRoot`